ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/components/style/ImageComponentStyle.kt

```diff
@@ -0,0 +1,90 @@
+package com.revenuecat.purchases.ui.revenuecatui.components.style
+
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.Immutable
+import androidx.compose.runtime.ReadOnlyComposable
+import androidx.compose.ui.graphics.Shape
+import androidx.compose.ui.layout.ContentScale
+import com.revenuecat.purchases.paywalls.components.properties.ColorInfo
+import com.revenuecat.purchases.paywalls.components.properties.Size
+import java.net.URL
+
+@Suppress("LongParameterList")
+@Immutable
+internal class ImageComponentStyle private constructor(
+    @get:JvmSynthetic
+    val visible: Boolean,
+    @get:JvmSynthetic
+    val url: URL,
+    @get:JvmSynthetic
+    val lowResURL: URL?,
+    @get:JvmSynthetic
+    val darkURL: URL?,
+    @get:JvmSynthetic
+    val darkLowResURL: URL?,
```

Comment:
I think we can consolidate these as well, to just keep the `url` and `lowResURL`. We have `ThemeImageUrls.urlsForCurrentTheme` to make sure they are the correct ones for the current theme.

Reply:
Ahhh nice! I also missed that 🤦 Thanks again!

Reply:
I'm just passing the `ThemeImageUrls` which already contain both the `web` and `lowResWebp` urls. Did the change in https://github.com/RevenueCat/purchases-android/pull/1959/commits/680f7c8418868de790c7c43b3d737d564e5aa492